go/coordinator/internal/logservice/grpc/record_log_service.go

```diff
@@ -0,0 +1,43 @@
+package grpc
+
+import (
+	"context"
+	"github.com/chroma/chroma-coordinator/internal/common"
+	"github.com/chroma/chroma-coordinator/internal/grpcutils"
+	"github.com/chroma/chroma-coordinator/internal/proto/logservicepb"
+	"github.com/chroma/chroma-coordinator/internal/types"
+	"github.com/pingcap/log"
+	"go.uber.org/zap"
+	"google.golang.org/protobuf/proto"
+)
+
+func (s *Server) PushLogs(ctx context.Context, req *logservicepb.PushLogsRequest) (*logservicepb.PushLogsResponse, error) {
+	res := &logservicepb.PushLogsResponse{}
+	collectionID, err := types.ToUniqueID(&req.CollectionId)
+	if err != nil {
+		log.Error("collection id format error", zap.String("collection.id", req.CollectionId))
+		return nil, common.ErrCollectionIDFormat
+	}
+	var recordsContent [][]byte
+	for _, record := range req.Records {
+		record.CollectionId = ""
+		data, err := proto.Marshal(record)
+		if err != nil {
+			log.Error("marshaling error", zap.Error(err))
+			grpcError, err := grpcutils.BuildInvalidArgumentGrpcError("records", "marshaling error")
+			if err != nil {
+				return nil, grpcError
+			}
+			return nil, err
```

Comment:
Should return a logservicepb.PushLogsResponse struct with proper field values.

Reply:
I think we should return error if it's error case... otherwise your grpc respons will have 200 status code

Reply:
Agree with @Ishiihara here

Reply:
I can go verify but I think the default interceptor is going to ignore the response anyways...

Reply:
and if we are going to return results in error case, we should be very specific about what we are returning. is the result partial?